unlock-app/src/hooks/useCrossChainRoutes.ts

```diff
@@ -122,6 +122,21 @@ export const useCrossChainRoutes = ({
       })),
   })
 
+  // Get tokenId for renewals
+  const { data: tokenId } = useGetTokenIdForOwner(
```

Comment:
I think we may already have that info in the context. Pleaase check how "normal" renewals access this data?